README.md

```diff
@@ -25,6 +25,26 @@ Most notably, the CI example includes sample configuration for the following pla
 
 On their own, these examples may not make a lot of sense. There's a lot more documentation below that should cover everything included in these examples and more.
 
+## Versioning v8.0.0 and beyond
+
+Rok8s-scripts contains a number of dependencies that have various ways of versioning themselves.  Most notably, helm tends to break backward compatibility with every minor release.  We have decided that post v8 of rok8s-scripts, we will update our versions according to the version change of the underlying tool.  For example, if helm changes from `2.13.0` to `2.14.0`, we will change the version of rok8s scripts by one minor version.  This will be clearly mentioned in the release notes. This means that a minor version of rok8s-scripts could introduce breaking changes to the CI/CD pipelines that are using it.
```

Comment:
I think Helm should be capitalized here. As a tiny personal preference, I also like single spaces to start a sentence, but that's far from necessary here.

Reply:
Fixed